<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>980</width>
    <height>400</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resources/resources.qrc">
    <normaloff>:/logos/tym-logo</normaloff>:/logos/tym-logo</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QSplitter" name="horizontalSplitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="0" colspan="5">
         <widget class="QTreeView" name="libraryView">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>400</width>
            <height>0</height>
           </size>
          </property>
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="alternatingRowColors">
           <bool>false</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>280</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Root:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="rootDirLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string notr="true">&lt;dir&gt;</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../resources/resources.qrc">
            <normaloff>:/action_icons/unexpand</normaloff>:/action_icons/unexpand</iconset>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../resources/resources.qrc">
            <normaloff>:/action_icons/expand</normaloff>:/action_icons/expand</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget1">
       <layout class="QGridLayout" name="rightPanel">
        <property name="spacing">
         <number>2</number>
        </property>
        <item row="1" column="0">
         <widget class="TrackInfosView" name="trackInfos" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QTableView" name="searchResultsView">
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <attribute name="horizontalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>21</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionImport"/>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <property name="separatorsCollapsible">
     <bool>true</bool>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionImport"/>
   <addaction name="actionRemove"/>
   <addaction name="separator"/>
   <addaction name="actionSearch"/>
   <addaction name="actionRename"/>
   <addaction name="actionExport"/>
   <addaction name="separator"/>
   <addaction name="actionSettings"/>
   <addaction name="actionLoginLogout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionImport">
   <property name="icon">
    <iconset resource="../../resources/resources.qrc">
     <normaloff>:/action_icons/import</normaloff>:/action_icons/import</iconset>
   </property>
   <property name="text">
    <string>Import</string>
   </property>
   <property name="toolTip">
    <string>Import files into library</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../../resources/resources.qrc">
     <normaloff>:/action_icons/about</normaloff>:/action_icons/about</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="../../resources/resources.qrc">
     <normaloff>:/action_icons/params</normaloff>:/action_icons/params</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="../../resources/resources.qrc">
     <normaloff>:/action_icons/close</normaloff>:/action_icons/close</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionSearch">
   <property name="icon">
    <iconset resource="../../resources/resources.qrc">
     <normaloff>:/action_icons/search</normaloff>:/action_icons/search</iconset>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
   <property name="toolTip">
    <string>Search on Beatport from files name's informations</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="icon">
    <iconset resource="../../resources/resources.qrc">
     <normaloff>:/action_icons/delete</normaloff>:/action_icons/delete</iconset>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
   <property name="toolTip">
    <string>Remove selected tracks from library</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionSetDefaultResult">
   <property name="icon">
    <iconset resource="../../resources/resources.qrc">
     <normaloff>:/action_icons/tick</normaloff>:/action_icons/tick</iconset>
   </property>
   <property name="text">
    <string>Set as better result</string>
   </property>
   <property name="toolTip">
    <string>Mark this result to use it for other tasks (rename, export, etc.)</string>
   </property>
  </action>
  <action name="actionSearchResultDelete">
   <property name="icon">
    <iconset resource="../../resources/resources.qrc">
     <normaloff>:/action_icons/close</normaloff>:/action_icons/close</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionRename">
   <property name="icon">
    <iconset resource="../../resources/resources.qrc">
     <normaloff>:/action_icons/rename</normaloff>:/action_icons/rename</iconset>
   </property>
   <property name="text">
    <string>Rename</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="icon">
    <iconset resource="../../resources/resources.qrc">
     <normaloff>:/action_icons/export</normaloff>:/action_icons/export</iconset>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
   <property name="toolTip">
    <string>Export selected tracks as Traktor NML collection</string>
   </property>
  </action>
  <action name="actionLoginLogout">
   <property name="text">
    <string notr="true">loginLogout</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TrackInfosView</class>
   <extends>QWidget</extends>
   <header>widgets/trackinfosview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resources/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>libraryView</receiver>
   <slot>expandAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>569</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>551</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>libraryView</receiver>
   <slot>collapseAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>603</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>575</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
